src/codeu/chat/server/Model.java

```diff
@@ -182,7 +182,7 @@ public void updateMessageLink(Message message) {
 	  // EDIT - Malik Graham
 	  // Updating the next message for the last message
       try {
-    	  ds.updateLastMessage(message);
+    	  ds.updateNextMessage(message);
```

Comment:
It throws an exception because we're making a call to the database, and whenever we do that a SQL Exception could possibly be thrown.